docs/supported.md

```diff
@@ -91,3 +93,60 @@ Earlier and Later versions might be only partially supported, we don't fully kno
 - _7_ - **build indicator**: If you're developing using `wrangler pages dev`, we do hard refreshes so the indicator doesn't appear. If you run your app locally using `next dev`, this option works fine.
 
 - _8_ - **Turbopack-specific options**: Turbopack is not currently supported on `@cloudflare/next-on-pages` (this might change in the future so we might reconsider the addition of this option).
+
+## Rendering and Data Fetching
+
+### Incremental Static Regeneration
+
+Incremental Static Regeneration (ISR) is a rendering mode in Next.js that allows you to automatically cache and periodically regenerate pages with fresh data. Next.js [does not support](https://nextjs.org/docs/pages/building-your-application/rendering/incremental-static-regeneration) building ISR pages for the edge runtime, and as such, pages should be changed to use server side rendering (SSR) instead.
+
+ISR pages are built by the Vercel CLI to something called prerender functions. These are Node.js Lambda functions that can be called in the background while serving the page from the cache. It is not possible to use these with Cloudflare Pages and they are not compatible with the edge runtime.
+
+next-on-pages will use the static fallback files for your ISR/prerendered pages.
+
+### Statically Generated Routes Edge Cases
+
+Next.js performs a form of static analysis on routes to determine whether they can be [statically generated](https://nextjs.org/docs/app/building-your-application/rendering/static-and-dynamic-rendering), or if they need to be dynamically rendered at request time. In most cases, next-on-pages can handle these pages fine. However, there are some edge cases where this is not possible with pages that have dynamic parameters.
+
+#### `generateStaticParams`
+
+When doing static site generation (SSG) in the app directory, you need to tell the page that it is not possible for non-prerendered dynamic routes to occur, i.e. [they should 404](https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamicparams). This is so that it doesn't generate a Node.js Lamba function for the dynamic parameter, while generating prerendered HTMl files for all params provided by `generateStaticParams`.
+
+```diff
++ export const dynamicParams = false;
+```
+
+#### `getStaticPaths`
+
+In the pages directory, when doing static site generation (SSG), there is a similar scenario as there was with `generateStaticParams`. Because of how ISR functions work, you need to tell the page that [it should 404](https://nextjs.org/docs/pages/api-reference/functions/get-static-paths#fallback-false) for non-pregenerated dynamic routes. This is done by setting `fallback: false` in `getStaticPaths`.
```

Comment:
```suggestion
In the pages directory, when doing static site generation (SSG), there is a similar scenario as there was with `generateStaticParams`. Because of how ISR functions work, you need to tell the page that [it should 404](https://nextjs.org/docs/pages/api-reference/functions/get-static-paths#fallback-false) for non-prerendered dynamic routes. This is done by setting `fallback: false` in `getStaticPaths`.
```